### PR TITLE
refactor: remove singleplayer

### PR DESCRIPTION
**Pull request by mieszko4**
Remove support for uploading singleplayer games and remove support for challenges.
If dev tries to upload non-multiplayer SDK then `Rune SDK script url must end with /multiplayer.js or /multiplayer-dev.js` err will show up.

### Test plan
- [x] Upload game with singleplayer.js. Make sure it shows new err
- [x] Upload game with multiplayer.js. Make sure it succeeds

### CODE REVIEW

**Review by mzbyszynski — COMMENTED** [review]
I haven't really looked at this repo at all yet. Might be good for @Gongreg to review? He's going to give me a crash course next week but that will be too late 🤓

**Comment by mieszko4** [comment]
@Gongreg Could you take a look?

**Comment by mieszko4** [comment]
I see @Gongreg is off today. @helios1138 Could you take a look?

File: packages/rune-games-cli/src/lib/validateGameFiles.ts [hunk 1]
```diff
@@ -113,35 +113,41 @@ export async function validateGameFiles(
...
+          semver.lt(sdkVersionCoerced, minSdkVersionCoerced)
+        ) {
+          errors.push({
+            message: `Rune SDK is below minimum version (included ${sdkVersion}, min ${minSdkVersion})`,
+          })
+        }
+      } else {
         errors.push({
-          message: `Rune SDK is below minimum version (included ${sdkVersion}, min ${minSdkVersion})`,
+          message: `Rune SDK must be multiplayer`,
```
Comment (helios1138) [inline packages/rune-games-cli/src/lib/validateGameFiles.ts]: Perhaps `Singleplayer Rune SDK is no longer supported`?

**Review by helios1138 — APPROVED** [review]
LGTM ✅, just had a minor wording suggestion

File: packages/rune-games-cli/src/lib/validateGameFiles.ts (hunk 1 above)
Comment (mieszko4) [reply packages/rune-games-cli/src/lib/validateGameFiles.ts]: In this PR I exclusively check if game is multiplayer.
So if dev includes any other file in `index.html`, e.g:
```html
<script src="https://cdn.jsdelivr.net/npm/rune-games-sdk@4.8.1/dist/libs.js"></script>
```
then showing `Singleplayer Rune SDK is no longer supported` would be confusing.

Comment (helios1138) [reply packages/rune-games-cli/src/lib/validateGameFiles.ts]: Then maybe let's say smth like `Rune SDK script url should end in /multiplayer.js`

Comment (mieszko4) [reply packages/rune-games-cli/src/lib/validateGameFiles.ts]: Sure, I will add `Rune SDK script url must end with /multiplayer.js or /multiplayer-dev.js`

Comment (mieszko4) [reply packages/rune-games-cli/src/lib/validateGameFiles.ts]: Done in https://github.com/rune/rune-games-sdk/pull/223/commits/e3db7655c8d9d55ae56ff37c61b9946ba5eb14b6

**Pull request merged**